### ISSUE TITLE
Context.js error when running Example in new project

### ISSUE THREAD

**Issue by talves** [peacechen/famous-carousel#2]
Error from `Context.js`

```
Uncaught TypeError: Cannot read property 'appendChild' of null
```

**Comment by talves** [comment]
Missed the selector in the html class.  My Bad :-1:

**Comment by peacechen** [comment]
The global.html example was referencing the debug build which I forgot to change back. Fixed in commit 6e79aa1. Thanks for catching that.

**Comment by talves** [comment]
Yeah, not a real issue. :+1:
